test/enums.jl

```diff
@@ -138,6 +138,10 @@ end
 @test repr(apple) == "apple::$(string(Fruit)) = 0"
 @test string(apple) == "apple"
 
+@enum UI8::UInt8 ten=0x0A el=0x03 t=0x07
```

Comment:
Can you move the new tests to line 155, i.e. below existing ones? Here they're in the middle of the  `Fruit` enum tests.

Reply:
Done!